### ISSUE TITLE
So AE models can't generate text right?

### ISSUE THREAD

**Issue by good74152** [varinf/TransformersDataAugmentation#6]
The paper uses 3 kinds of pre-trained models, AR and Seq2Seq both have fine-tuned and text generation phase, so the AE models just only need pre-trained?
And how the AE models generates text? Input like this [y_i, x_1, <mask>, ...., x_m]?

**Comment by varunentropy** [comment]
For AE models data generation, we simply use the pre-training strategy. You simply mask a few words in [y_i, x_1, <MASK>, ...., x_m]  and then predict those words, this will give you an augmented sentence.
